### PR TITLE
Java skeleton

### PR DESCRIPTION
**Pull request by maggieolaya**
I restructured my files so that they match up with the java implementation branch. I also fixed some java syntax errors in my naming and in my code. This week I began looking into the web service aspect of the service and am figuring out which classes I want to have interacting with it. I also am in the midst of finding equivalent java imports to the ones in the GO files and figuring out which classes need which.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,6 +56,58 @@
   </properties>
 
   <dependencies>
```
Comment (markpatton) [inline pom.xml:58]: Some of the variables used for versions are not defined.

File: src/main/java/org/eclipse/pass/download/Main.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package org.eclipse.pass.manuscript;
```
Comment (markpatton) [inline src/main/java/org/eclipse/pass/download/Main.java]: Is there a need for a command line application? What does it do?

File: src/main/java/org/eclipse/pass/download/Serve.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package org.eclipse.pass.manuscript;
```
Comment (markpatton) [inline src/main/java/org/eclipse/pass/download/Serve.java]: Name the class something like DownloadServlet. Also make the "/lookup" to match the existing service and your documentation.

**Review by markpatton — CHANGES_REQUESTED** [review]
Make sure all the class names are in camel case starting with an uppercase letter.
Make sure all method names are in camel case starting with a lowercase letter.

Add .DS_Store to .gitignore and remove those files from git.

Add a github action to build and test the service. Base it on an existing one like pass-doi-service .github/workflows/ci.yml. (Just copy the file to the right place and edit it slightly.)

Make sure "mvn verify" completes successfully. The GitHub action will run it for you when you push commits.

**Comment by maggieolaya** [comment]
Created class to hold information about the manuscripts, obtained from DOI lookup. Working on building and writing JSON files. Added more to servlet outline.

**Comment by maggieolaya** [comment]
I believe I am finished with designing my service and ready to move to implementation. I am having trouble with the checkstyle errors, they don't show up on my end but only show up when I create the pull request.

**Comment by markpatton** [comment]
@maggieolaya 

The checkstyle errors should show up if you run
```
mvn verify
```

File: src/main/java/org/eclipse/pass/manuscript/Unpaywall.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * 
+ * @author Maggie Olaya
+ */
+
+
+public class Unpaywall {
+  /**
+   * Unpaywall lookup service.
+   */
+  public Manuscript[] lookup(String doi) {
```
Comment (markpatton) [inline src/main/java/org/eclipse/pass/manuscript/Unpaywall.java]: May want to return a List<Manuscript> instead, but an array is fine too.

**Review by markpatton — CHANGES_REQUESTED** [review]
Look good! Just need to get the checkstyle errors fixed and I'll merge.

**Review by markpatton — APPROVED** [review]

**Pull request merged**